Dockerfile

```diff
@@ -0,0 +1,34 @@
+FROM google/cloud-sdk:284.0.0-alpine
+
+ARG BUILD_DATE
+ARG VCS_REF
+
+RUN echo $BUILD_DATE
+RUN echo $VCS_REF
+
+LABEL org.label-schema.build-date=$BUILD_DATE \
+      org.label-schema.vcs-ref=$VCS_REF \
+      org.label-schema.vcs-url="https://github.com/mintel/k8s-mysqldump.git" \
+      org.label-schema.schema-version="1.0.0-rc1" \
+      org.label-schema.name="k8s-gcloud-export" \
+      org.label-schema.description="An image for exporting mysql databases using 'gcloud sql export', and pushing them to a Google bucket." \
+      org.label-schema.vendor="Mintel Group Ltd." \
```

Comment:
org.label-schema has been deprecated in favour of org.opencontainers.image. If we're gonna only provide one it should probably be the latter or, better, provide both - see https://github.com/mintel/gce-proxy/blob/master/Dockerfile for one I did yesterday.

Reply:
Cool will do

Reply:
I dropped usage of label-schema just to avoid duplication.

I think we'll be fine to take this approach going forward too - we don't rely on external tools to inspect our docker images (and I'm sure k8s / dockerhub will support opencontainers)